mscore/musescore.cpp

```diff
@@ -1316,7 +1316,12 @@ MuseScore::MuseScore()
             cornerLabel->setPixmap(QPixmap(":/data/mscore.png"));
             cornerLabel->setGeometry(width() - 48, 0, 48, 48);
             }
-      QString result = layoutInput();
+      QSettings s;
+      if (!s.contains("firstStartup")) {
+            QString keyboardLayout = layoutInput();
+            s.setValue("firstStartup", true);
+            s.setValue("keyboardLayout", keyboardLayout);
```

Comment:
Yes. I have made that small change. I think that wherever we need to read that setting, we can just get it from QSettings.